gobblin-service/src/test/resources/template_catalog/multihop/jobTemplates/distcp-push-hdfs-to-gridfs.template

```diff
@@ -0,0 +1,45 @@
+# ====================================================================
+# Job configurations
+# ====================================================================
+job.name=DistcpToGridFS
+
+#team.name and dataset.name to be supplied via flowConfig
+from=/data/out/${dataset.name}
```

Comment:
are the from and to the same because we are making a copy from hdfs to gridfs?

Reply:
it's just the local cluster path in this scenario, so it doesn't matter what the fs is as it gets accounted for later.